*** tests/A-080-080-M-ok.out	2023-10-19 17:10:34.321632000 +0100
--- tests/A-080-080-M-ok.outhyp	2023-10-26 03:47:36.108376203 +0100
***************
*** 23,29 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 23,29 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 32,39 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 53;1|49
! 54;1|43
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 32,40 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 53;1|
! 54;1|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
