*** tests/A-108-108-M-ok.out	2023-10-19 17:08:13.278813000 +0100
--- tests/A-108-108-M-ok.outhyp	2023-10-25 18:02:11.709943387 +0100
***************
*** 18,24 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
--- 18,24 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
  3 - Copiar
***************
*** 27,32 ****
--- 27,34 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|
  Menu de Edição
  1 - Visualizar
***************
*** 37,43 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;2|'abc
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 39,45 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;2|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
